src/misc/lv_anim.c

```diff
@@ -402,13 +397,16 @@ static void anim_timer(lv_timer_t * param)
 
         /*If the linked list changed due to anim. delete then it's not safe to continue
          *the reading of the list from here -> start from the head*/
-        if(anim_list_changed)
+        if(anim_list_changed) {
             a = _lv_ll_get_head(&LV_GC_ROOT(_lv_anim_ll));
-        else
+        }
+        else {
+            a->last_timer_run = lv_tick_get();
```

Comment:
I'd move it right below ` uint32_t elaps = lv_tick_elaps(a->last_timer_run);`
Else when the list changes for any reason `last_timer_run` won't be updated.